framework/wazuh/agent.py

```diff
@@ -444,76 +444,52 @@ def _remove_manual(self, backup=False, purge=False):
         :return: Message.
         """
 
-        # Get info from DB
+        # Check if agent exists
         self._load_info_from_DB()
 
         f_keys_temp = '{0}.tmp'.format(common.client_keys)
-        open(f_keys_temp, 'a').close()
-
-        f_keys_st = stat(common.client_keys)
-        chown(f_keys_temp, common.ossec_uid, common.ossec_gid)
-        chmod(f_keys_temp, f_keys_st.st_mode)
 
-        f_tmp = open(f_keys_temp, 'w')
         agent_found = False
-        with open(common.client_keys) as f_k:
+        with open(common.client_keys) as f_k, open(f_keys_temp, 'w') as f_tmp:
```

Comment:
What are exactly f_k and f_tmp? I find this code a bit confusing due to lack of comments and/or too cryptic var names.

Reply:
`f_k` means _file keys_ and `f_tmp` means _temporary file_, referring to the temporary file created before replacing the original one. I can improve those variable names.

Reply:
Fixed in https://github.com/wazuh/wazuh/pull/2966/commits/b5d1f4353507caae5a2f3c597626bb302d873d64